### ISSUE TITLE
Could not find a part of the path error

### ISSUE THREAD

**Issue by Moostical** [KSP-CKAN/NetKAN#9861]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Operating System

Windows

### CKAN Version

1.33.2

### Games

KSP 1

### Game Version

1.12.3

### Describe the bug

Every time I try to install RO/RP-1 I get this error message 

### Steps to reproduce

Retry the installation

### Relevant log output

```
Could not find a part of the path 'c:\program files (x86)\steam\SteamApps\common\Kerbal Space Program\Ships\@thumbs\SPH\E42.png'.
```

**Comment by JonnyOThan** [comment]
There's a missing backslash before `@thumbs` but maybe markdown ate it

**Comment by HebaruSan** [comment]
Thanks for the report. This is fixed in KSP-CKAN/CKAN#3893.

**Comment by HebaruSan** [comment]
Hey @Moostical, thanks for the report. I tried installing FerramAerospaceResearchContinued (the mod that installs that file) just now, and it seems to work fine:

![image](https://github.com/KSP-CKAN/NetKAN/assets/1559108/65cbf13c-ee0d-4347-bca5-9e14e2d68e79)

So this is probably caused by some peculiarity on your system rather than a core problem with CKAN or the metadata (otherwise other people would see the same problem). We'll need to know more about what's going on on your PC to help. First, can you check whether the `c:\program files (x86)\steam\SteamApps\common\Kerbal Space Program\Ships` folder exists?
